README.md

```diff
@@ -141,6 +141,9 @@ Once completed, a browser will automatically open with the results.
 The following commands are available:
  * `storybook:build`: Builds the static storybook assets
  * `storybook:serve` Builds storybook assets and then serves them locally at `localhost:6006`
+ * `storybook:deploy` Builds storybook assets and then serves them to [github pages](https://blockchain.github.io/blockchain-wallet-v4-frontend). **You will probably need to run `cd ./packages/blockchain-info-components && git remote add origin git@github.com:blockchain/blockchain-wallet-v4-frontend.git` first.**
+
```

Comment:
What does your `git remote -v` look like?

Mine is:
origin	git@github.com:blockchain/blockchain-wallet-v4-frontend.git (fetch)
origin	git@github.com:blockchain/blockchain-wallet-v4-frontend.git (push)

Reply:
It was unset within the blockchain-info-components directory for me so i had to set it there. In the root mine looks like yours.